iterating-over-data/practice-iterating-over-data.md

```diff
@@ -0,0 +1,465 @@
+# Practice: Iterating Over Data
+
+## Directions
+
+Complete all questions below.
+
+## Practice
+
+<!-- Question 1 -->
+<!-- prettier-ignore-start -->
+### !challenge
+
+* type: multiple-choice
+* id: EJew-RW0t
+* title: Which Range is Correct?
+
+##### !question
+
+Given a function called `stopwatch` that takes in the argument `22`, which of the following loops would  correctly start at `1` and increment all the way up to *and include* `22`?
+
+##### !end-question
+
+##### !options
+
+* `for i in 22:`
+* `for i in range(22)`
+* `for i in range(1,22)`
+* `for i in range(1,23)`
+
+##### !end-options
+
+##### !answer
+
+`for i in range(1,23)`
```

Comment:
Adding bullet point for format consistency

```suggestion
* `for i in range(1,23):`
```

@audreyandoy FYI, when changing options, the answer should also be updated!